bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/RuleManagerImpl.java

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+ * Copyright (c) 1997, 2015 by ProSyst Software GmbH
+ * http://www.prosyst.com
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    ProSyst Software GmbH - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.smarthome.automation.core;
+
+import java.util.Set;
+
+import org.osgi.framework.BundleContext;
+
+/**
+ * @author Yordan Mihaylov - Initial Contribution
+ *
+ */
+public class RuleManagerImpl extends RuleManager {
```

Comment:
I wonder if this class could be somehow removed and merged with the RuleRegistryImpl? See comments about storeRules and loadRules.